service/cloud/gcp_streamer_internal_test.go

```diff
@@ -12,14 +12,16 @@
 // License for the specific language governing permissions and limitations under
 // the License.
```

Comment:
Funny. It matches the header so the diff thinks that the payload file was not deleted but renamed and repurposed completely.